### PR TITLE
EES-646 - summary/headline block add bugfix (block is added as expected)

### PR DESCRIPTION
**Pull request by bennettstuart**
The content for these sections was being stored locally in the component parent. So I have added a useEffect to watch for content changes and will update the local state if it changes.

### CODE REVIEW

**Comment by benoutram** [comment]
@ntsim is on holiday. I sent him an e-mail to remind him about this change when he returns.

File: src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx [hunk 1]
```diff
@@ -33,6 +33,10 @@ const PublicationReleaseContent = () => {
     EditableBlock[]
   >(release.summarySection.content);
 
+  useEffect(() => {
+    setSummarySectionBlocks(release.summarySection.content);
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx:37]: This is the wrong way to go about this. Pass `release.summarySection.content` directly to the `EditableSectionBlocks`.

File: src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseHeadlines.tsx [hunk 2]
```diff
@@ -29,6 +29,10 @@ const ReleaseHeadlines = ({ release }: Props) => {
     release.headlinesSection.content,
   );
 
+  useEffect(() => {
+    setHeadlineBlocks(release.headlinesSection.content);
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseHeadlines.tsx:33]: Same as above

File: src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx (hunk 1 above)
Comment (bennettstuart) [reply src/explore-education-statistics-admin/src/modules/find-statistics/PublicationReleaseContent.tsx:37]: I thought as much, but I think you removed this as part as some performance improvement - so I didn't want to undo that. I'll fix up as you've suggested, thanks

**Pull request merged**